.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Before creating the pull request, please make sure you do the following:
+
+- Read the Contributing Guidelines at https://github.com/AmadeusITGroup/AgnosUI/blob/main/CONTRIBUTING.md.
+- Check that there isn't already a PR that solves the problem the same way. If you find a duplicate, please help us reviewing it.
+- Update the corresponding documentation if needed.
+- Include relevant tests that fail without this PR but pass with it.
+
```

Comment:
Shouldn't we include some basic tasks in the template?
For example title well formatted, issue linked, context, explanation of the solution, which packages include